src/main/scala/com/campudus/tableaux/router/auth/permission/RoleModel.scala

```diff
@@ -218,9 +212,27 @@ class RoleModel(jsonObject: JsonObject) extends LazyLogging {
         })
 
         editCellValueJson
+      }
 
-      } else {
-        isActionAllowed(EditCellValue)
+      objects.columnOpt match {
+        case Some(column) => {
+
+          column.languageType match {
+            case MultiLanguage => {
+              val langtags = objects.tableOpt.map(_.langtags.getOrElse(Seq.empty[String])).getOrElse(Seq.empty[String])
+
+              getCellValueJson(langtags)
+            }
+
+            case MultiCountry(countryCodes) => {
+              getCellValueJson(countryCodes.codes)
+            }
+
+            case _ => isActionAllowed(EditCellValue)
+          }
+        }
+
+//        case _ =>
```

Comment:
Delete commented code

Reply:
forgot to uncomment, prevents warning "match may not be exhaustive."